app.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var hoursOfOperation = ['6am: ', '7am: ', '8am: ', '9am: ', '10am: ', '11am: ', '12pm: ', '1pm: ', '2pm: ', '3pm: ', '4pm: ', '5pm: ', '6pm: ', '7pm: ', '8pm: ', 'Total: '];
+var hoursOfOperation = ['Store Location', '6:00am', '7:00am', '8:00am', '9:00am', '10:00am', '11:00am', '12:00pm', '1:00pm', '2:00pm', '3:00pm', '4:00pm', '5:00pm', '6:00pm', '7:00pm', '8:00pm', 'Daily Location Total'];
```

Comment:
You may want to have only the times in this array. Then just add the 'Store Location' and 'Total' cells before and after your loop in the `makeHeaderRow` function.

It is better practice to have only related data in an array, e.g. only times.